pkg/fakeoperator/fakeoperator.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2019 The KubeCarrier Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fakeoperator
+
+import (
+	"fmt"
+
+	"github.com/go-logr/logr"
+	"github.com/spf13/cobra"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/healthz"
+
+	fakev1alpha1 "github.com/kubermatic/kubecarrier/pkg/apis/fake/v1alpha1"
+	"github.com/kubermatic/kubecarrier/pkg/fakeoperator/internal/controllers"
+	"github.com/kubermatic/kubecarrier/pkg/internal/util"
+)
+
+type flags struct {
+	metricsAddr          string
+	enableLeaderElection bool
+	healthAddr           string
```

Comment:
Can you please move `healthAddr` next to/after `metricsAddr`? Just want to group similar flags together.